src/modules/conda.rs

```diff
@@ -20,6 +20,9 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     if config.ignore_base && conda_env == "base" {
         return None;
     }
+    if config.ignore_pixi_envs && context.get_env("PIXI_ENVIRONMENT_NAME").is_some() {
+        return None;
+    }
```

Comment:
pixi sets `CONDA_DEFAULT_ENV` as well thus we need to deactivate the conda module in this case